cli/run_util_test.go

```diff
@@ -515,3 +519,138 @@ workflows:
 	results, err := runWorkflowWithConfiguration(time.Now(), "target", config, []envmanModels.EnvironmentItemModel{})
 	require.Equal(t, 1, len(results.StepmanUpdates))
 }
+
+func TestInitializeStepDir(t *testing.T) {
+	tempTestResultsDir, err := pathutil.NormalizedOSTempDirPath("testing")
+	if err != nil {
+		t.Fatalf("failed to create testing dir, error: %s", err)
+	}
+
+	if err := os.Setenv("BITRISE_TEST_RESULTS_DIR", tempTestResultsDir); err != nil {
```

Comment:
Consider more different names for BITRISE_TEST_RESULTS_DIR and BITRISE_TEST_RESULT_DIR